climakitae/explore/vulnerability.py

```diff
@@ -291,7 +288,9 @@ class CavaParams(param.Parameterized):
     heat_idx_threshold = param.Number(
         default=None, allow_None=True, doc="Threshold for heat index"
     )
-    one_in_x = param.Integer(default=None, allow_None=True, doc="1-in-x year event")
+    one_in_x = param.Parameter(
+        default=None, allow_None=True, doc="1-in-x year event(s) — number or list"
```

Comment:
doc here reinforces my comment about passing in a list not an np.array of a list